source/manual/errors.html.md

```diff
@@ -8,9 +8,50 @@ last_reviewed_on: 2020-05-18
 review_in: 6 months
 ---
 
-## How we categorise errors
+## How we serve errors on GOV.UK
 
-Sometimes applications will encounter exceptions. This policy describes what we should do for different types of errors. It was first proposed in [RFC 87](https://github.com/alphagov/govuk-rfcs/blob/master/rfc-087-dealing-with-errors.md).
+When a request to GOV.UK fails, we need to handle the error in some way, so that GOV.UK does not look completely broken. Rather than embedding this into every app, we have multiple layers of error handling.
+
+1. **CDN**. There are a few scenarios here:
+
+  - When the origin servers return a 5xx response:
+
+    - For [GET or HEAD requests](https://github.com/alphagov/govuk-cdn-config/blob/d77313abdb5098e2b350de7a0992375e50ff03a3/vcl_templates/www.vcl.erb#L339), we:
+      - Try to [serve the page from a sequence of static mirrors](https://github.com/alphagov/govuk-cdn-config/blob/d77313abdb5098e2b350de7a0992375e50ff03a3/vcl_templates/www.vcl.erb#L274)
+      - Serve [a basic inline error page](https://github.com/alphagov/govuk-cdn-config/blob/d77313abdb5098e2b350de7a0992375e50ff03a3/vcl_templates/www.vcl.erb#L531) if [all the mirrors fail](https://github.com/alphagov/govuk-cdn-config/blob/d77313abdb5098e2b350de7a0992375e50ff03a3/vcl_templates/www.vcl.erb#L363) (uh oh!)
+
+    - For [other requests](https://github.com/alphagov/govuk-cdn-config/blob/d77313abdb5098e2b350de7a0992375e50ff03a3/vcl_templates/www.vcl.erb#L367), we:
+      - [Serve the response as-is from the origin](https://github.com/alphagov/govuk-cdn-config/blob/d77313abdb5098e2b350de7a0992375e50ff03a3/vcl_templates/www.vcl.erb#L374) (see below)
+
+  - When the origin servers are offline, we:
+      - [Skip normal request handling and begin error handling](https://varnish-cache.org/docs/trunk/users-guide/vcl-built-in-subs.html#vcl-backend-error)
+
+      - Try to [serve the page from a sequence of static mirrors](https://github.com/alphagov/govuk-cdn-config/blob/d77313abdb5098e2b350de7a0992375e50ff03a3/vcl_templates/www.vcl.erb#L274)
+      - Serve [a basic inline error page](https://github.com/alphagov/govuk-cdn-config/blob/d77313abdb5098e2b350de7a0992375e50ff03a3/vcl_templates/www.vcl.erb#L531) if [all the mirrors fail](https://github.com/alphagov/govuk-cdn-config/blob/d77313abdb5098e2b350de7a0992375e50ff03a3/vcl_templates/www.vcl.erb#L363) (uh oh!)
```

Comment:
The two points above link to the same code as the identical points in the section above it. I think it could be simplified / easier to parse:

|Before | After |
|-------|------|
|<img width="652" alt="Screenshot 2020-05-28 at 14 17 48" src="https://user-images.githubusercontent.com/5111927/83146372-11d42d80-a0ee-11ea-9667-878811b777c3.png">|<img width="684" alt="Screenshot 2020-05-28 at 14 17 04" src="https://user-images.githubusercontent.com/5111927/83146316-fec15d80-a0ed-11ea-8c17-ee3f8b338a9b.png">|

Reply:
Sure, that makes sense, updated. My original thinking was that they could change independently.